### ISSUE TITLE
Instancing relatively large Viewport Textures consistently causes frame drops.

### ISSUE THREAD

**Issue by KnightNine** [godotengine/godot#40250]
**Godot version:**
v3.2.1

**OS/device including version:**
win10, gtx 1080, GLES3

**Issue description:**
Consistent freeze when instancing a scene with a `ViewportTexture` with a high resolution, despite the scene being preloaded.
I also noticed that during these frame drops, the Profiler registers it as being below the average frame rate during them, as if the profiler itself is also freezing.
![image](https://user-images.githubusercontent.com/34152090/87115352-d027be00-c227-11ea-8279-8318fe341be1.png)

**Steps to reproduce:**
create a scene with a 3d viewport, same as this [demo project](https://github.com/godotengine/godot-demo-projects/tree/3.0-d69cc10/viewport/gui_in_3d), set the Viewport size to something like 3500x3500 or higher.

Then create another scene where your previous scene is preloaded into it, and add a control or something to instance the scene into this scene at runtime in order to see the effect it has on framerate.

**Minimal reproduction project:**
I don't want to spend time on one right now in case this is not an issue but just a known limitation in Godot. So let me know and I will.

**Comment by Zireael07** [comment]
Most likely shaders compiling?

**Comment by KnightNine** [comment]
@Zireael07 do they have to compile for every instance? There's no way to preload them?

**Comment by Zireael07** [comment]
Nope, GLES 2 and 3 don't allow preloading shaders. There are some tricks, @BastiaanOlij I think showcased some, but IIRC they don't work for viewport textures...

**Comment by KnightNine** [comment]
Darn, guess this isn't a bug report as much as it is a performance issue, i could try and instance this 3d viewport scene from a thread and just have a delay before the instance appears rather than a stutter if that's possible.

**Comment by Calinou** [comment]
This is a known issue. It makes [dynamic resolution](https://github.com/Calinou/godot-dynamic-resolution-demo) unfortunately not viable. It might be solvable in Vulkan though: https://github.com/godotengine/godot-proposals/issues/917
